Libraries/Lists/ListView/ListView.js

```diff
@@ -259,13 +259,46 @@ var ListView = React.createClass({
   },
 
   /**
+   * This function is a pass through for the ScrollView#scrollTo
    * Scrolls to a given x, y offset, either immediately or with a smooth animation.
    *
-   * See `ScrollView#scrollTo`.
+   * DEPRECATED PARAMS:
+   * This function can take the arguments (x:number, y:number, animated:boolean)
+   * but it is deprecated, so please use an object {x:number, y:number, animated:boolean}
+   *
+   * Example:
+   *
+   * `scrollTo({x: 0, y: 0, animated: true})`
    */
-  scrollTo: function(...args: Array<mixed>) {
+  scrollTo: function(
+    y?: number | { x?: number, y?: number, animated?: boolean },
+    x?: number,
+    animated?: boolean
+  ) {
     if (this._scrollComponent && this._scrollComponent.scrollTo) {
-      this._scrollComponent.scrollTo(...args);
+      var scrollToProps = {};
+
+      // The correct params were passed so use them
+      if (typeof y === 'object') {
```

Comment:
hi - it was  more that when you look at this function, the params were ..args array of anything.  My goal was to make this method signature clear and concise.